### PR TITLE
Add admin UI

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by e0ipso** [comment]
Good stuff. Are you thinking about any particular structure for the form or just dumping the variables in there?

**Comment by amitaibu** [comment]
Honestly, my _real_ plan is the @BurningDog who opened this issue in #248 will pick this up ;)

**Comment by amitaibu** [comment]
Ok, I'll be working on this, as I want to get the formatter configurable via UI

restful___site-install

**Comment by amitaibu** [comment]
`admin/config/services/restful`

restful___site-install

Ready for review.

File: restful.admin.inc [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    '#type' => 'radios',
+    '#title' => t('Default formatter'),
+    '#description' => t('Determine the default formatter that would be used.'),
+    '#options' => $options,
+    '#default_value' => variable_get('restful_default_output_formatter', 'hal_json'),
+    '#required' => TRUE,
+  );
+
+  // Add description for each formatter.
+  foreach (restful_get_formatter_plugins() as $plugin_name => $plugin) {
```
Comment (e0ipso) [inline restful.admin.inc:24]: Can we have just one loop instead of two over `restful_get_formatter_plugins()`?

File: restful.module [hunk 2]
```diff
@@ -181,6 +181,17 @@ function restful_menu() {
     $items[$base_path . '/login']['access callback'] = 'user_is_anonymous';
   }
 
+  // Add administration page.
+  $items['admin/config/services/restful'] = array(
+    'title' => 'RESTful',
+    'description' => 'Administer the RESTful module.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('restful_admin_settings'),
+    // @todo: Have our own administer permission?
```
Comment (e0ipso) [inline restful.module:190]: Agreed.

**Comment by e0ipso** [comment]
I am going to merge this and address my own comments in a follow up PR.

**Comment by e0ipso** [comment]
Merged in https://github.com/Gizra/restful/commit/265e8433f0102270aea994f0400460d8263371cc

**Comment by BurningDog** [comment]
Ok wow, that's a lot cooler than I had in mind :)

**Pull request closed without merging**